File: Dubi-C#/Clases/ProductoGenerico.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Clases
{

    public class ProductoGenerico
    {
        private int id;
        private string nombre;
        private string descripcion;
        private string color;
        private double precio;
        private double stockActual;
        private TipoProductoG tipo;
        private UnidadDeMedida unidad;
        private int Activo;

        public ProductoGenerico(int id, string nombre, string descripcion, string color, double precio, double stockActual, TipoProductoG tipo, UnidadDeMedida unidad, int activo)
        {
            this.id = id;
            this.nombre = nombre;
            this.descripcion = descripcion;
            this.color = color;
            this.precio = precio;
            this.stockActual = stockActual;
            this.tipo = tipo;
            this.unidad = unidad;
            Activo = activo;
        }

        public int Id { get => id; set => id = value; }
        public string Nombre { get => nombre; set => nombre = value; }
        public string Descripcion { get => descripcion; set => descripcion = value; }
        public string Color { get => color; set => color = value; }
        public double Precio { get => precio; set => precio = value; }
        public double StockActual { get => stockActual; set => stockActual = value; }
        public TipoProductoG Tipo { get => tipo; set => tipo = value; }
        public UnidadDeMedida Unidad { get => unidad; set => unidad = value; }
        public int Activo1 { get => Activo; set => Activo = value; }
    }
}

File: Dubi-Java/src/Modelo/ProductoGenerico.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package Modelo;

/**
 *
 * @author Katherine Espinoza 20122127
 */
public class ProductoGenerico {    
    private int id;    
    private String nombre;
    private String descripcion;
    private String color;
    private double precio;
    private double stockActual;
    private TipoProductoG tipo;
    private UnidadDeMedida unidad;
    private int Activo;

    public ProductoGenerico() {
    }
    
    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getDescripcion() {
        return descripcion;
    }

    public void setDescripcion(String descripcion) {
        this.descripcion = descripcion;
    }

    public String getColor() {
        return color;
    }

    public void setColor(String color) {
        this.color = color;
    }

    public double getPrecio() {
        return precio;
    }

    public void setPrecio(double precio) {
        this.precio = precio;
    }

    public double getStockActual() {
        return stockActual;
    }

    public void setStockActual(double stockActual) {
        this.stockActual = stockActual;
    }

    public int isActivo() {
        return Activo;
    }

    public void setActivo(int Activo) {
        this.Activo = Activo;
    }

    public TipoProductoG getTipo() {
        return tipo;
    }

    public void setTipo(TipoProductoG tipo) {
        this.tipo = tipo;
    }

    public UnidadDeMedida getUnidad() {
        return unidad;
    }

    public void setUnidad(UnidadDeMedida unidad) {
        this.unidad = unidad;
    }
}
